classes/class-wc-connect-nux.php

```diff
@@ -186,11 +217,14 @@ public function set_up_nux_notices() {
 			}
 
 			$jetpack_install_status = $this->get_jetpack_install_status();
+			$banner_to_display = $this->get_banner_type_to_display( array(
```

Comment:
The reason I suggested merging (or alternatively calling `get_jetpack_install_status()` directly from `get_banner_type_to_display()`) is that the `$status` object you're creating and passing is only used by `get_banner_type_to_display()`. 

It seems to me like you might as well fetch the jetpack state and the two options directly from `get_banner_type_to_display`.